website/content/docs/installation.md

```diff
@@ -170,6 +176,14 @@ bloopAggregateSourceDependencies in Global := true
 
 <span class="label warning">Note</span> that `bloopAggregateSourceDependencies` has to be scoped globally.
 
+### Maven
+
+To generate Bloop's configuration using Maven, run (from your project directory):
+
+```sh
+$ mvn ch.epfl.scala:maven-bloop_2.12:1.0.0-M10:bloopInstall
```

Comment:
You can use the same trick we did here to get the latest version automatically:
https://github.com/scalacenter/bloop/pull/489/files#diff-85567f8b3a7a1207fae1d03eb8722217R153

Reply:
I'll fix this myself!

Reply:
Thanks, I did look briefly at that snippet but thought it was just formatting.

BTW, the `:1.0.0-M10` part can be skipped altogether, but I'm not sure how Maven resolves the version then: if I try this now it downloads `1.0.0-M10+8-b6694b98`.

Reply:
I'll add that detail to the docs. It may be useful to those that want to use latest master release.